src/core/sys/posix/sys/ioctl.d

```diff
@@ -339,6 +333,11 @@ else version (OSX)
         ushort ts_yyy;
     }
 
+    enum uint TIOCGSIZE = TIOCGWINSZ;
+    enum uint TIOCSSIZE = TIOCSWINSZ;
+
+    import core.sys.posix.sys.filio; // File related ioctls
```

Comment:
Same reason as above (following OSX .h structure). People include "filio.h" if they need the file specific IOCTL defines. So in D, the would need to include the filio.d module and the ioctl.d module.

I am new to D, so maybe I am missing something related to modules that you are pointing out -- would it be problematic for people to include both, say, filio.d and ioctl.d if they want access to the file related ioctls?